src/components/sheet/sheet-add-retailer.vue

```diff
@@ -66,6 +72,7 @@ const submit = async () => {
     };
     emit("back");
   } catch (err: any) {
+    await store.retailer.set(BulletState.NULL)
```

Comment:
If there is an error, it should be set to BulletState.Error not Null

Reply:
Changed